### ISSUE TITLE
0.2.73 test fails

### ISSUE THREAD

**Issue by ndowens** [Exiv2/exiv2#1379]
**Describe the bug**
tiff_test fails, which I put the error in Additional context section

**To Reproduce**
Steps to reproduce the behaviour:
1. Provide the image with which you observed the issue (commonly named PoC)
2. Provide exact command to reproduce the issue:
  make test
3. Mention the branch/commit in which you observed the issue (i.e: `master` or `0.27-maintenance`) 
  0.2.73 release

**Expected behavior**
Tests to pass

**Desktop (please complete the following information):**
 - OS: [e.g. Linux, iOS, Windows] Linux
 - Compiler & Version [Gcc 7.3, Clang-6, MSVC 2017] gcc 8.3.0
 - Compilation mode and/or compiler flags

**Additional context**
ERROR: test_run (tiff_test.test_tag_compare.OutputTagExtract)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ndowens/adelie-packages/user/exiv2/src/exiv2-0.27.3-Source/tests/system_tests.py", line 654, in test_run
    self.compare_stdout(i, command, processed_stdout, stdout)
  File "/home/ndowens/adelie-packages/user/exiv2/src/exiv2-0.27.3-Source/tests/tiff_test/test_tag_compare.py", line 102, in compare_stdout
    self.pa_data = self.parse_pa(got_stdout.splitlines())
  File "/home/ndowens/adelie-packages/user/exiv2/src/exiv2-0.27.3-Source/tests/tiff_test/test_tag_compare.py", line 30, in parse_pa
    exif, image, tag = tmp[0].split('.')
ValueError: not enough values to unpack (expected 3, got 1)

**Comment by clanmills** [comment]
Thank You for reporting this.   This is a remarkable find, because I've never seen this and the CI runs the test suite on every commit on several linux distributions.

I've reproduced something like this with 0.27.3 tag on Ubuntu.  However I find the message coming from:

```
FAIL: test_run (bugfixes.github.test_issue_247.TestFirstPoC)
```

You're reporting this from tiff_test which runs OK for me.

```bash
81 rmills@ubuntu:~/gnu/github/exiv2/0.27-maintenance/tests $ python3 runner.py --verbose tiff_test/test_tiff_test_program.py 
test_run (test_tiff_test_program.TestTiffTestProg) ... ok

----------------------------------------------------------------------
Ran 1 test in 0.013s

OK
82 rmills@ubuntu:~/gnu/github/exiv2/0.27-maintenance/tests $ 
```

All test pass on the current 0.27-maintenance.  So, this has been fixed (although I'm not aware any relevant change).  I'm very surprised that this is in the released code.  I'll investigate on Tuesday.

**Comment by clanmills** [comment]
I've performed the following without finding a case to answer:

1 I inspected the log from the Linux build bundle https://exiv2.org/builds/exiv2-0.27.3-Linux64.tar.gz
- the test suite ran without incident during the build performed on 2020-06-30

2 I downloaded and built/tested the source bundles for v0.27.3
- from Exiv2.org: https://exiv2.org/builds/exiv2-0.27.3-Source.tar.gz
- from GitHub: https://github.com/Exiv2/exiv2/releases/tag/v0.27.3
- Both built and ran the test suite without incident

I don't think there's anything seriously wrong here.  To make progress, I would have to investigate the Linux distro and version of bash/python/gcc/gtest.  I'm busy writing a book about Exiv2 and would like to focus on that task.  https://clanmills.com/exiv2/book
